apis/core/v1alpha1/backupconfiguration_types.go

```diff
@@ -458,6 +458,11 @@ const (
 	TypeSchedulerEnsured      = "SchedulerEnsured"
 	ReasonSchedulerNotEnsured = "SchedulerNotEnsured"
 	ReasonSchedulerEnsured    = "SchedulerEnsured"
+
+	// TypeInitialBackupTriggered indicates whether the backup is triggered or not.
```

Comment:
// TypeInitialBackupTriggered indicates whether the initial backup is triggered or not.